app/grants/management/commands/sent_cart_reminder.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+"""Define the Grant subminer management command.
+
+Copyright (C) 2020 Gitcoin Core
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published
+by the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+from datetime import datetime
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.db.models import Max, F
+from django.utils import timezone
+
+from dashboard.utils import get_tx_status, has_tx_mined
+from grants.clr import predict_clr
+from grants.models import Contribution, Grant, CartActivity, Subscription
+from grants.views import clr_active
+from marketing.mails import warn_subscription_failed, remember_your_cart
+from townsquare.models import MatchRound
+
+
+class Command(BaseCommand):
+    help = 'Sent reminder to user who forgot its cart '
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--test',
+            default=False,
+            type=bool,
+            help="Only process and display the carts to being delivered"
+        )
+        parser.add_argument(
+            '--full-cart',
+            default=False,
+            type=bool,
+            help="Should the cart being delivered partially"
+        )
+
+    def handle(self, *args, **options):
+        mr = MatchRound.objects.current().first()
+
+        if mr:
+            valid_to = mr.valid_to
+            hours = int((valid_to - datetime.now()).total_seconds() / 3600)
+            if hours != 72:
```

Comment:
i think ill want to send this manually at a couple diff intervals so  u can remove this check